### PR TITLE
Add module name to ModuleDoc rule errors

### PR DESCRIPTION
**Pull request by jacknoble**
https://github.com/lpil/dogma/issues/92

### CODE REVIEW

File: lib/dogma/rules/module_doc.ex [hunk 1]
```diff
@@ -43,11 +43,12 @@ defmodule Dogma.Rules.ModuleDoc do
...
   end
 
-  defp check_node({:defmodule, m, [_, [do: module_body]]} = node, errors) do
+  defp check_node({:defmodule, m, [mod, [do: module_body]]} = node, errors) do
+    {_,_,names} = mod
     if module_body |> moduledoc? do
       {node, errors}
     else
-      {node, [error( m[:line] ) | errors]}
+      {node, [error( m[:line], Enum.join(names, ".") ) | errors]}
```
Comment (lpil) [inline lib/dogma/rules/module_doc.ex]: Can we use `Module.concat/1` please? :)

**Comment by lpil** [comment]
Looks great, thank you. Just one little note above :)

File: lib/dogma/rules/module_doc.ex (hunk 1 above)
Comment (jacknoble) [inline lib/dogma/rules/module_doc.ex]: Ah, good point. One thing is that
```
iex(1) [:ModName] |> Module.concat |> to_string
"Elixir.ModName"
```

That 'Elixir' prefix in the errror message seems a little noisy but if it doesn't bother you I'll change it.

Comment (lpil) [inline lib/dogma/rules/module_doc.ex]: Ah, I didn't think about that. What you've done is better then. :)

**Comment by lpil** [comment]
Coverage dropped due to bug with detecting string interpolation.

**Pull request merged**